### ISSUE TITLE
Folders and files are not displayed at the file tree

### ISSUE THREAD

**Issue by mariosimao** [microsoft/vscode-remote-release#1038]
Issue Type: <b>Bug</b>

Sometimes when connecting to remote, the file tree is empty. Just need to close the connection and reconnect. Is not a big problem, but after several times, its is a little bit frustating.

Extension version: 0.44.2
VS Code version: Code 1.36.1 (2213894ea0415ee8c85c5eea0d0ff81ecc191529, 2019-07-08T22:56:38.504Z)
OS version: Darwin x64 18.6.0
Remote OS version: Linux x64 3.10.0-862.11.6.el7.x86_64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|16.00GB (0.39GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: stage|
|OS|Linux x64 3.10.0-862.11.6.el7.x86_64|
|CPUs|Intel Core Processor (Haswell, no TSX, IBRS) (4 x 2394)|
|Memory (System)|7.64GB (0.63GB free)|
|VM|0%|
</details>

**Comment by roblourens** [comment]
Are you sure the window successfully connected when you see this? If you look in the lower left corner, does it say "Disconnected from..."?

Or if you switch to a different viewlet like search, and switch back to the file tree, then do you see it?

**Comment by mariosimao** [comment]
Yes, I am sure that the window successfully connected. The integrated terminal works fine, I can open files, edit and save them. When this bug happens to me, only the file tree does not work.

Switching views does not solve the problem. The file tree appears again only if I disconnect and connect again.

Is important to say that the bug only happens when I start a connection, it never happened during a current connection to remote.

**Comment by isidorn** [comment]
@mariosimao can you reproduce this with latest vscode insiders https://code.visualstudio.com/insiders/
Also can you provide some reproducable steps so we can investigate more

**Comment by mariosimao** [comment]
Since it is something that happens randomly, I could not reproduce it on the VSCode Insiders.
Probably it current fixed there. Just to let you know the steps I took to find the bug on the VSCode in production:

1. On my iTerm2, logged in to remote (with SSH).
2. Open remote on VSCode. I did not have to write my password because I was already logged in on my local terminal.
3. File tree view does not show anything (see image bellow)

![Screenshot 2019-08-01 at 16 48 45](https://user-images.githubusercontent.com/24276493/62303815-1e133e00-b47d-11e9-800c-86eaae4e9bcc.png)

PS: I use `"remote.SSH.showLoginTerminal": true` on my user settings.

Since I could not reproduce it on the insiders version, I will close this issue.
